masschallenge/django-accelerator
# [AC-6660] Staff unable to add or view subnav associations

### Changes introduced in [AC-6660](https://masschallenge.atlassian.net/browse/AC-6660):
- Add node subnav association permissions for the mc app

### How to test
- While on this branch on django-accelerator run `make migrate` on impact to run the migration
- Login with the credentials email: `39765-user@example.com` password: `password`
- Visit: http://localhost:8181/admin/fluent_pages/page/324/change/?language=en-us
- Notice that the user does not see the node subnavassociation section

- In a new browser window login as demo admin and go to http://localhost:8181/admin/simpleuser/user/39765/change/
- Grant the user all Node Sub Navigation Association permissions for both mc and accelerator (rem to save) 
- Go back to the browser with user 39765-user@example.com and refresh the fluent admin page. 
- You should now be able to see the node subnavassociation section

**Test7 QA**
-  this has been deployed to [test7](https://test7.masschallenge.org)
- One of the affected users was Corinne (corinne@masschallenge.org)
- Visit the admin page for the [user](https://test7.masschallenge.org/admin/simpleuser/user/59439/) and notice she has add/delete/change permissions for Node Sub Navigation for the mc app
- To test this you'll have to actually login as the user rather than masquerade. This is because you'd still see the field as a superuser while masquerading
- This can be accomplished by changing the user's password and logging in as them (accomplished by ssh-ing into the test machine while testing)
- View any admin page for a fluent page and notice that you can see the node subnavigation section
- Logged in as yourself remove the Node Sub Navigation permissions from the user then log in as the user and notice the she can no longer see the the node subnavigation section on the admin page for a fluent page
